### PR TITLE
Style: Format strings using f-strings in app/api/resources/user.py

### LINKED ISSUE #739: Use f-strings on User Resources file
## Description
As a developer,
I need change strings interpolation in User Resources file,
so that I can make that file follow Coding Standards.

Format strings using f-strings when appropriate for `app/api/resources/user.py` file.

## Mocks

As an example, here:

https://github.com/anitab-org/mentorship-backend/blob/9102175e85302a04ad4d3bc4d1f89f30f840b0f9/app/api/resources/user.py#L171-L172

We can use f-strings such as:

```
@users_ns.response(HTTPStatus.CREATED, f"{messages.PASSWORD_SUCCESSFULLY_UPDATED})
```

### Update [Required]
- [ ] Update all string instances to use f-string, when applicable for `app/api/resources/user.py` file

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by Sanchit-Trivedi**
### Description

Used f-strings(Min. Requirement :Python 3.6) instead of old %-formatting
in user.py file in mentorship-backend/app/api/resources directory.

Fixes #739 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)

### How Has This Been Tested?

Tested using existing tests by running
`python -m unittest discover tests`

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: app/api/resources/user.py
Comment (SanketDG) [inline app/api/resources/user.py]: couple of extra spaces in the end

**Review by SanketDG — DISMISSED** [review]
Looks good to me apart from a few extra spaces, feel free to fix those

File: app/api/resources/user.py
Comment (Sanchit-Trivedi) [reply app/api/resources/user.py]: Done

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
```

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@Sanchit-Trivedi you did a good job here :)
I just added a few style comments for extra empty lines that can be removed, after you change those I will approve.
Thank you for your contribution 🎉

**Comment by Sanchit-Trivedi** [comment]
@isabelcosta thanks for the feedback. Have made the requested changes.

File: app/api/resources/user.py
Comment (isabelcosta) [reply app/api/resources/user.py]: @Sanchit-Trivedi I think you missed this one ;)

**Comment by Sanchit-Trivedi** [comment]
@isabelcosta Pardon my silly error.  It's fixed now

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
@users_ns.response(HTTPStatus.CREATED, f"{messages.PASSWORD_SUCCESSFULLY_UPDATED}")
```

File: app/api/resources/user.py
Comment (isabelcosta) [inline app/api/resources/user.py]:
```suggestion
@users_ns.response(HTTPStatus.BAD_REQUEST, f"{messages.USER_ENTERED_INCORRECT_PASSWORD}")
```

**Comment by isabelcosta** [comment]
@Sanchit-Trivedi I just noticed 2 nitpick things after that I can approve ;) I asked for more help as well :)

File: app/api/resources/user.py
Comment (Sanchit-Trivedi) [reply app/api/resources/user.py]: @isabelcosta Made the requested changes

**Review by isabelcosta — DISMISSED** [review]
Great work @Sanchit-Trivedi 👏 
Thank you for your contribution! 🎉

**Review by vj-codes — APPROVED** [review]
LGTM 🎉

File: app/api/resources/user.py
Comment (devkapilbansal) [inline app/api/resources/user.py:426]: Shouldn't we use a single line here, as done above

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1: all strings are changes as requested

File: app/api/resources/user.py
Comment (vj-codes) [reply app/api/resources/user.py:426]: Since you suggested the change and approved the PR too , do you want this change  and label will be updated as `Status changes requested` or I can update the label as `Ready to merge` ? @devkapilbansal

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
Changing the status of review @vj-codes

File: app/api/resources/user.py
Comment (devkapilbansal) [reply app/api/resources/user.py:426]: I think we should make this change. If this needs to be done, then label should be changes requested, else it is ready for merging. What are your thoughts on this??

**Comment by vj-codes** [comment]
@devkapilbansal you mean requesting changes right ?

**Comment by devkapilbansal** [comment]
> @devkapilbansal you mean requesting changes right ?

Yes 😅

**Comment by vj-codes** [comment]
@Sanchit-Trivedi please have a look at the changes requested and update accordingly .Thanks!

**Review by isabelcosta — COMMENTED** [review]
@Sanchit-Trivedi can you please fix the merge conflicts in the PR, and then I can approve it again :)

**Comment by Sanchit-Trivedi** [comment]
I currently have a medical emergency at home will do it once free. Thanks

**Comment by isabelcosta** [comment]
@Sanchit-Trivedi that's ok! Thank you for the update and stay safe! 🙏

**Review by vj-codes — CHANGES_REQUESTED** [review]
@Sanchit-Trivedi hey please solve the merge conflicts .Thanks!

**Comment by Sanchit-Trivedi** [comment]
@vj-codes @isabelcosta Really sorry for the delay. Totally slipped off my mind. I have resolved the conflicts. Please have a look.
Thanks

**Review by isabelcosta — DISMISSED** [review]
No worries about the delay.
Great work @ Sanchit-Trivedi 👏 Thank you for your contribution!

**Review by vj-codes — CHANGES_REQUESTED** [review]
@Sanchit-Trivedi hey there are some merge conflicts again , please can you fix them ? This PR can be merged after that :)

**Comment by Sanchit-Trivedi** [comment]
@vj-codes @isabelcosta Please have a look

**Review by isabelcosta — DISMISSED** [review]
🤗 Great work!

**Review by devkapilbansal — DISMISSED** [review]
LGTM :+1:

**Comment by isabelcosta** [comment]
@devkapilbansal @anitab-org/qa-team can anyone test this PR please?

**Review by gaurivn — DISMISSED** [review]
Screenshot (745)
Screenshot (746)
Screenshot (747)
Screenshot (749)

Looks perfect to me, thank you

**Comment by gaurivn** [comment]
@Sanchit-Trivedi, could you pls resolve merge conflicts

**Comment by devkapilbansal** [comment]
@Sanchit-Trivedi please resolve merge conflicts and also squash your commits into one

**Review by vj-codes — CHANGES_REQUESTED** [review]
@Sanchit-Trivedi hey there are still some merge conflicts , please resolve them .Then this PR is good to go!

**Comment by devkapilbansal** [comment]
@Sanchit-Trivedi are you still looking into this?

**Comment by Sanchit-Trivedi** [comment]
I was busy with my endsems. But I have resolved the merge conflicts a few number of times. By the time the PR is approved new merge conflicts arise.

**Comment by devkapilbansal** [comment]
> I was busy with my endsems. But I have resolved the merge conflicts a few number of times. By the time the PR is approved new merge conflicts arise.

@Sanchit-Trivedi No problem take your time. Actually to merge any PR it needs to be reviewed and tested so that any existing feature is not affected and the PR solves the issue as describes. 
Update it when you have some time

**Comment by devkapilbansal** [comment]
Also, please squash your commits in one and rebase your branch to current develop

**Comment by isabelcosta** [comment]
@Sanchit-Trivedi is part of the review process, other PRs move on a little more quickly and then merge conflicts naturally arise. We understand you may be busy. Let us know when you fix the merge conflicts, please.
cc @devkapilbansal

File: app/api/resources/user.py
Comment (devkapilbansal) [inline app/api/resources/user.py:87]: @Sanchit-Trivedi it should be `HTTPStatus.UNAUTHORIZED.value`. Same for other places too.

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
Use black to format your code as the black tests are failing.

**Comment by vj-codes** [comment]
@Sanchit-Trivedi please resolve the merge conflicts and update with the changes requested above , thanks!

**Comment by vj-codes** [comment]
@Sanchit-Trivedi closing due to inactivity
Thank you for your contribution :)

**Pull request closed without merging**
